tests/uccsd_test.py

```diff
@@ -32,6 +32,28 @@ def test_He_uccsd_vqe_exact(self):
         Egs = Egs_elec + Enuc
         self.assertLess(abs(Egs-Efci), 1.0e-11)
 
+    def test_He_uccsd_vqe_exact(self):
+        print('\n')
+        # The FCI energy for He atom in a cc-pvdz basis
+        Efci = -2.887594831090938
+        # The Nuclear repulsion energy
+        Enuc =  0.0
```

Comment:
Line 40 could be removed and the following additional modifications should be introduced:
line 53: Egs_elec -> Egs
line 54: remove

The same modifications could be made to the following test as well, for consistency. On a side note, I believe that line 68 should be removed; since in this test we are using psi4 to generate the one- and two-electron integrals, we should not be reading from a file.